components/ChargeBar/ChargeBar.jsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import {
+  components,
+  spacing as themeSpacing,
+  baseFontSize as defaultBaseFontSize,
+} from '../shared/theme';
+import { hexToRgba } from '../shared';
+import Icon from '../Icon';
+
+const Bar = styled.div`
+  border-radius: 4px;
+  overflow: hidden;
+  position: relative;
+
+  ${({ width, background, spacing: { xxsmall } }) => `
+    background-color: ${hexToRgba(background, 0.39)};
+    height: 4px;
```

Comment:
Looking at figma, I notice that this guys In the desktop version it's a little bigger, it has a height of 8px. What do you think?